### PR TITLE
Add thorax_kidneys example

### PR DESCRIPTION
**Pull request by psavery**
This is a cinematic rendering example which uses the new light scattering
model that was recently added to VTK.

It does not include interactions since rendering is slow.

If the data file does not exist in the current directory, it is
downloaded automatically and placed in the current directory.

### CODE REVIEW

**Review by cjh1 — APPROVED** [review]
LGTM

File: Examples/Python/cinematic_rendering/thorax_kidneys.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+    print("Download complete")
+
+# Volumetric shadow settings
+volumetric_scattering_blending = 2
+global_illumination_reach = 0.35964
+
+# Read the data
+reader = vtk.vtkXMLImageDataReader()
+reader.SetFileName("thorax_kidneys.vti")
```
Comment (aylward) [inline Examples/Python/cinematic_rendering/thorax_kidneys.py:30]: Should  use data_file var as arg to SetFileName

**Review by aylward — CHANGES_REQUESTED** [review]

File: Examples/Python/cinematic_rendering/thorax_kidneys.py (hunk 1 above)
Comment (psavery) [reply Examples/Python/cinematic_rendering/thorax_kidneys.py:30]: Good point!

Comment (psavery) [reply Examples/Python/cinematic_rendering/thorax_kidneys.py]: Fixed

**Pull request merged**